src/main/java/com/redhat/cloud/notifications/NotificationsApp.java

```diff
@@ -5,17 +5,36 @@
 
 import javax.enterprise.event.Observes;
 
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
 public class NotificationsApp {
+
     private static final String BUILD_COMMIT_ENV_NAME = "OPENSHIFT_BUILD_COMMIT";
     private static final String BUILD_REFERENCE_ENV_NAME = "OPENSHIFT_BUILD_REFERENCE";
     private static final String BUILD_NAME_ENV_NAME = "OPENSHIFT_BUILD_NAME";
 
+    public static final String FILTER_REGEX = ".*(/health(/\\w+)?|/metrics) HTTP/[0-9].[0-9]\" 200.*\\n?";
+    private static final Pattern pattern = Pattern.compile(FILTER_REGEX);
+
     private static final Logger LOG = Logger.getLogger(NotificationsApp.class);
 
+    // we do need a event as parameter here, otherwise the init method won't get called.
     void init(@Observes StartupEvent ev) {
+        filterAccessLogs();
+
         showVersionInfo();
     }
 
+    private void filterAccessLogs() {
```

Comment:
Perhaps rename to something like "init access log filter"   ?

Reply:
I don't have a strong opinion about the name. I am fine with both, so I will take your suggestion.